styles.css

```diff
@@ -0,0 +1,55 @@
+header {
+    background-color: white;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    height:8px;
+}
+.main {
+  display: flex;
+  flex-direction: column;
+  padding: 2em 20vw;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```

Comment:
You have used 'Lucida Sans' here but you are not providing this font in your html file. So most users won't have it but you have lots of fallbacks so this is not a problem just making sure you know.

Reply:
I see! Thanks for pointing out.